lib/Bootstrap.php

```diff
@@ -125,6 +125,8 @@ public static function bootstrap()
             $loader = include __DIR__ . '/../../../../vendor/autoload.php';
         } elseif (getenv('PIMCORE_PROJECT_ROOT') != '' && file_exists(getenv('PIMCORE_PROJECT_ROOT') . '/vendor/autoload.php')) {
             $loader = include getenv('PIMCORE_PROJECT_ROOT') . '/vendor/autoload.php';
+        } elseif (defined('PIMCORE_PROJECT_ROOT') && file_exists(PIMCORE_PROJECT_ROOT . '/vendor/autoload.php')) {
```

Comment:
This doesn't seem to be related, can you please revert this change?

Reply:
Oops, I think I just made it worser.

I simply copied the file from the master.

I'll look at it again after the break. Sorry!

Reply:
I don't know why the file is still shown under changes, it is identical to the file from the master.

I'm checking the merge confliction now.